docs/modding/suitmodding/addinglocals.md

```diff
@@ -0,0 +1,186 @@
+To improve clarity and structure in the article, I’ll organize the information in a more logical flow. The goal will be to avoid overwhelming the reader with too much technical detail upfront and to guide them step-by-step, with each section focusing on one aspect of the process. Here's a revised version:
+
+---
+
+# Part 2: Localisation Overrides
```

Comment:
Part 2?

Reply:
I'll just remove the parts maybe. The point was to smooth the reader from the adding parts to the next articles